esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -4122,10 +4155,14 @@ void CWsWuFileHelper::sendWUComponentLogStreaming(CHttpRequest* request, CHttpRe
         throw makeStringException(ECLWATCH_INVALID_INPUT, "Empty Name detected");
 
     IEspContext* ctx = request->queryContext();
+    double version = ctx->getClientVersion();
     Owned<CWULogFileRequest> espRequest = new CWULogFileRequest(ctx, "WsWorkunits", request->queryParameters(), request->queryAttachments());
 
     WUComponentLogOptions options;
-    readWUComponentLogOptionsReq(espRequest->getFileOptions(), options);
+    if (version < 1.92)
```

Comment:
question: Should this also be checking a minimum version?  I suspect it doesn't need to.

Reply:
The code here is for File_ComponentLog which was added at version 1.90. Before that version, the File_ComponentLog does not exist and no client will use the code here. So, there is no need to check the minimum version for backward compatible.